docs/src/main/sphinx/release/release-439.md

```diff
@@ -0,0 +1,17 @@
+# Release 439 (7 Feb 2024)
+
+## General
+
+* Fix failure when setting session properties for a catalog with a `.` in its
+  name. ({issue}`20474`)
+* Fix potential query failure when writing to ORC files. ({issue}`20587`)
+
+## Hive connector
+
+* Fix query failure when a value has not been specified for the
+  `orc_bloom_filter_fpp` configuration property. ({issue}`16589`)
```

Comment:
`orc_bloom_filter_fpp` is a table property, not config property. Same for Iceberg connector.